doc/syntax.md

```diff
@@ -128,29 +451,67 @@ Currenly only spike output is supported.
 
    `process_spikes()`
 
-   concepts for refractoriness
+   emit_spike(): calling this function in the `update`-block results in firing
+   a spike through the neuron.
+### Concepts for refractoriness
+In order to model refractory and non-refractory states, two variables are
+necessary. First variable `t_ref` defines the duration of the refractoriness period.
```

Comment:
Add "The" in front of "First" and replace "refractoriness period" by "refractory time".